### PR TITLE
add Suwayomi tracker

### PR DESCRIPTION
**Pull request by AriaMoradi**

### CODE REVIEW

File: app/src/main/java/eu/kanade/tachiyomi/data/track/tachidesk/Tachidesk.kt [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+import okhttp3.Dns
+import okhttp3.OkHttpClient
+import eu.kanade.domain.manga.model.Manga as DomainManga
+import eu.kanade.domain.track.model.Track as DomainTrack
+
+class Tachidesk(private val context: Context, id: Long) : TrackService(id), NoLoginTrackService, EnhancedTrackService {
+    override val client: OkHttpClient =
+        networkService.client.newBuilder()
+            .dns(Dns.SYSTEM) // don't use DNS over HTTPS as it breaks IP addressing
+            .build()
```
[the remark this answers is not in the document]
Comment (jobobby04) [reply app/src/main/java/eu/kanade/tachiyomi/data/track/tachidesk/Tachidesk.kt:23]: Nvm I see its going to be handled in the API class

**Comment by arkon** [comment]
Is there still an initiative to rename Tachidesk? This seems like it'd make users think that it's an official desktop version of Tachiyomi, which it isn't.

**Comment by AriaMoradi** [comment]
> Is there still an initiative to rename Tachidesk? This seems like it'd make users think that it's an official desktop version of Tachiyomi, which it isn't.

We had a vote last year to rename it, Suwayomi came out as the second winner and we decided to not rename the app but name the org Suwayomi. Also I have never advertised Tachidesk as an official Tachiyomi app and have never seen anyone treat it that way in out discord server not sure how anyone could come to that conclusion.

**Comment by arkon** [comment]
> I'm not sure how anyone could come to that conclusion

I've seen people on both Reddit and Discord that were under that impression, and putting it in the app itself will only further perpetuate that false association. I'm a bit confused as to how you can possibly think no one would come to that conclusion given the naming.

File: app/src/main/java/eu/kanade/presentation/more/settings/screen/SettingsTrackingScreen.kt [hunk 2]
```diff
@@ -198,7 +198,7 @@ class SettingsTrackingScreen : SearchableSettings {
                             if (hasValidSourceInstalled) {
                                 trackManager.suwayomi.loginNoop()
                             } else {
-                                context.toast(context.getString(R.string.enhanced_tracking_warning, context.getString(trackManager.suwayomi.nameRes())), Toast.LENGTH_LONG)
+                                context.toast(context.getString(R.string.enhanced_tracking_warning, R.string.tracker_tachidesk), Toast.LENGTH_LONG)
```
Comment (Soitora) [inline app/src/main/java/eu/kanade/presentation/more/settings/screen/SettingsTrackingScreen.kt:201]: Why did you add back Tachidesk naming?

Comment (AriaMoradi) [reply app/src/main/java/eu/kanade/presentation/more/settings/screen/SettingsTrackingScreen.kt:201]: Referring to Extension?

Comment (AriaMoradi) [reply app/src/main/java/eu/kanade/presentation/more/settings/screen/SettingsTrackingScreen.kt:201]: It is referring to the extension and it's named Tachidesk.
